src/store/fs-chunk-data-store.ts

```diff
@@ -115,10 +115,23 @@ export class FsChunkDataStore implements ChunkDataStore {
         `${process.cwd()}/${chunkHashPath}`,
       );
 
-      await fs.promises.symlink(
-        targetPath,
-        this.chunkDataRootPath(dataRoot, relativeOffset),
-      );
+      const sourcePath = this.chunkDataRootPath(dataRoot, relativeOffset);
+
+      if (fs.existsSync(sourcePath)) {
```

Comment:
This looks backwards to me. Wouldn't we want to unlink the target? Also, why do anything in this case. If the symlink is already there we shouldn't need to replace it.

Reply:
now modified to just reduce the log verbosity undir this somewhat common and harmless condition